src/main/java/com/yahoo/identity/services/session/LoggedInSession.java

```diff
@@ -0,0 +1,21 @@
+package com.yahoo.identity.services.session;
+
+import com.yahoo.identity.services.account.Account;
+import com.yahoo.identity.services.credential.Credential;
+
+import javax.annotation.Nonnull;
+
+public interface LoggedInSession {
+
+    @Nonnull
+    String getUsername();
+
+    @Nonnull
+    Credential getCredential();
+
+    @Nonnull
+    Account getAccount();
+
+    @Nonnull
+    LoggedInSession sessionAccountUpdate(@Nonnull Account account);
```

Comment:
I think it should look like
```
AccountUpdate newAccountUpdate()
```
So caller can obtain the AccountUpdate to update the account.

Reply:
Emmm. That seems a little bit unexpected for me. My initial though: 
`AccountService` is not designed to be exposed to the `AccountsApiImpl`-> `AccountUpdate`/`AccountCreate` is better to be handled in `SessionImpl` or `LoggedInSessionimpl`, but not exposed to the upper level.

Reply:
The highest level's call is that now: `loggedInSession.sessionAccountUpdate(account).getCredential().toString()`, where can be called continuously when `sessionAccountUpdate(account)` returns `LoggedInSessionimpl`. If `AccountUpdate` is returned, I guess it will be exposed and we dont want to see that?

Reply:
I think AccountService shouldn't be exposed because it allow updating an account, getting account without permission check.
So that means another class has to do the permission check, is pass, then forward the request to AccountService.
So I think letting `LoggedInSession` return a `AccountUpdate` is okay since `LoggedInSession` guarantee that callers can only update the account corresponding to the `LoggedInSession`.
And I think an `Account` instance should represent an account that really exists, so caller shouldn't instantiate an `AccountImpl` by himself.